packages/manager/modules/telecom-dashboard/src/identity-check/form/cancellation/cancellation.controller.js

```diff
@@ -0,0 +1,44 @@
+export default class IdentityCheckFormCancellationCtrl {
+  constructor($translate, IdentityCheckService, TucToast) {
```

Comment:
missing /* @ngInject */
```suggestion
/* @ngInject */
  constructor($translate, IdentityCheckService, TucToast) {
```